telemetry_package/main.py

```diff
@@ -1,31 +1,140 @@
-import sys
+'''
+This code gets too many messages all at once. Hence it needs to: 
+    - The Node only sends the first message it receives
```

Comment:
What do you mean by that?

Reply:
This praticular error comes from the fact that I did not figure out a way to better implement the callback. I'l try and explain: 

Due to the callback function sending the messages on the websocket in an infinite loop, the Node does not refresh the message which is supposed to send. Hence, at the time I wrote the comment, the Node kept sending the same message over and over, while not updating which messages it got from the /rosout topic. 

The approach of reducing the callback to a minimum and executing it only if the websocket is active is a good idea I intend to implement. That would in fact solve the problem of the same message being sent by my node via websocket, since checking if the socket is still active gives the node a condition to update the message that needs to be sent.